v3/plugins/scalameta/metals/1.7.2/meta.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.7.2
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://rawcdn.githack.com/OlegIlyenko/scala-icon/c453c6f67dfcc1dbf0ca1e437d32d04f1d710983/scala-icon.svg
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-01-24"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:8-0cfbacb"
+      name: vscode-metals
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/scalameta/vsextensions/metals/1.7.2/vspackage/
```

Comment:
Microsoft forbids downloading extensions from the vscode marketplace when they're not used in VS Code (see https://github.com/eclipse/che/issues/14573). From looking at the github repo, it seems they don't publish vsix releases, so I don't know the best option here.

Reply:
temporary workaround provided -- I'll talk to the upstream developers about it

Reply:
Alternatively, if you can't get them to publish to GitHub or some other location, you could ask @mkuznyetsov  about how to build the vsix + source tarball and publish to download.jboss.org where we have all the 3rd party and RH-built extensions used in Che and CRW.